.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Is this really what we want?  It's currently `precise`, AIUI.

After reading the blog completely, I'm confused:
- per Travis job warning we have: `This job ran on our Precise, sudo: required environment which will be updated on May 3rd. You can check more details about this in our blog.` -- that's what sent me into the panic mode
- per the blog post we have: `We’re focusing on making the switch to our default stable image from Ubuntu Precise 12.04 to Ubuntu Trusty 14.04 in early Q3, sometime in July.`